src/collections/_documentation/clients/react-native/ram-bundles.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Using RAM Bundles'
+---
+
+The [RAM bundle](https://facebook.github.io/react-native/docs/performance#ram-bundles-inline-requires) format is a new approach to packaging React Native apps that optimizes your app's startup time. With RAM bundles, it is possible to load to memory only those modules that are needed for specific functionality, and only when needed.
+
+All the existing RAM bundle formats are explained in detail in the [Metro Bundler documentation](https://facebook.github.io/metro/docs/en/bundling).
+
+## Enabling RAM Bundles
+
+To enable RAM bundles for your app, please consult with the [official React Native documentation](https://facebook.github.io/react-native/docs/performance#enable-the-ram-format).
+
+If you use the official `react-native-sentry` integration of version `0.43.1` or newer, no additional actions are required. `sentry-cli`, which is used internally by the integration, will detect the bundle type automatically, and then upload the extracted modules to Sentry.
+
+## Uploading Bundles Manually
+
+Starting from version `1.43.0`, `sentry-cli` provides two additional parameters to the `upload-sourcemaps` command in order to simplify bundle uploads: `--bundle` and `--bundle-sourcemap`. Using those parameters, you can pass the path to the application bundle, along with its source map, and the bundle will be automatically extracted before the upload, if necessary:
+
+```sh
+sentry-cli releases files RELEASE_ID upload-sourcemaps --bundle main.bundle --bundle-sourcemap main.bundle.map
+```
+
+All bundle types (plain, Indexed RAM bundles, and File RAM bundles) are supported.
+
+## RAM Bundles and Performance
+
+The initial Sentry support for RAM bundles, added in `sentry-cli 1.43.0` and `react-native-sentry 0.43.1`, is known to have performance issues when handling RAM bundles with a large number of modules. This will be fixed in future client versions.
```

Comment:
I would mention here that this means upload performance, not runtime performance.

Reply:
Updated